docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,338 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_led_support:
+
+***********************************************
+Stock LED Support (White and RGB LED Control)
+***********************************************
+All Vertiq Gen 2 modules support use of our LED add-on board available with Vertiq's Pro Kit. For information about purchasing a Pro Kit, or for information about 
+purchasing more LED add-on boards, please contact us at info@vertiq.co. Please refer to your module's family page to see if it belongs to Gen 1 or Gen 2.
+
+Hardware Configuration
+=========================
+If you are using Vertiq's Pro Kit, simply plug the LED connector into the LED board, and you can move on to *Controlling Your LEDs*. Otherwise continue reading.
+
+Vertiq's Gen 2 modules all feature solder pads that allow you to easily connect with external power sources and other peripherals. The LED peripheral board connects 
+with your module's GPIO2 and GPIO3 connections where GPIO2 controls the RGB LED and GPIO3 controls the white LEDs. Depending on the size of your module, 
+you will see the GPIO values printed as either IOX or GPIOX where X is the GPIO value.
```

Comment:
This is only true on the solder pad CN boards, but not the downward eject CN boards which only have pin numbers.

Reply:
Do you have any downward eject boards?